### ISSUE TITLE
Questions about the input shape of the training parameters of the AER model

### ISSUE THREAD

**Issue by 943fansi** [sintel-dev/Orion#389]
In aer.py line 147-160, what is the shape of X array and y array? I am trying to use single `aer.py` to train and detect series anomalies.
My data is two columns of values: one with sequence values and one with labels.
X shape (4000, ), y shape (4000, ). What should be the input and output shapes passed into the AER model? Missing necessary parameter information, help is desperately needed.

        Args:
            X (ndarray):
                N-dimensional array containing the input sequences for the model.
            y (ndarray):
                N-dimensional array containing the output sequences we want to reconstruct.
        
        if y is None:
            y = X.copy()  # Reconstruct the same input.

        X = X[:, 1:-1, :]
        ry, y, fy = y[:, 0], y[:, 1:-1], y[:, -1]

model defined as follow:

    layers_encoder = [
                    {
                        "class": "tensorflow.keras.layers.Bidirectional",
                        "parameters": {
                            "layer": {
                                "class": "tensorflow.keras.layers.LSTM",
                                "parameters": {
                                    "units": "lstm_units",
                                    "return_sequences": False
                                }
                            },
                            "merge_mode": "concat"
                        }
                    }
                ]
    layers_decoder = [
                    {
                        "class": "tensorflow.keras.layers.RepeatVector",
                        "parameters": {
                            "n": "repeat_vector_n"
                        }
                    },
                    {
                        "class": "tensorflow.keras.layers.Bidirectional",
                        "parameters": {
                            "layer": {
                                "class": "tensorflow.keras.layers.LSTM",
                                "parameters": {
                                    "units": "lstm_units",
                                    "return_sequences": True
                                }
                            },
                            "merge_mode": "concat"
                        }
                    },
                    {
                        "class": "tensorflow.keras.layers.TimeDistributed",
                        "parameters": {
                            "layer": {
                                "class": "tensorflow.keras.layers.Dense",
                                "parameters": {
                                    "units": 1
                                }
                            }
                        }
                    }
                ]
    model = AER(layers_encoder=layers_encoder, layers_decoder=layers_decoder,
                 optimizer="tensorflow.keras.optimizers.Adam")
    model.fit(x_train)

**Comment by 943fansi** [comment]
I sovled.

**Comment by xuchunyu123** [comment]
I also have the same confusion, how did you solve it? Looking forward to your reply
